server/lorax_server/utils/layers.py

```diff
@@ -32,6 +32,7 @@
     HAS_EXLLAMA = False
 try:
     from lorax_server.utils.gptq.exllama import Ex4bitLinear
+    from lorax_server.utils.gptq.exllamav2 import QuantLinear as exllamav2QuantLinear
```

Comment:
Since we're still building the v1 kernels, would it make sense to fallback to v1 if v2 is unavailable?

```
HAS_EXLLAMA_V2 = True
try:
    from lorax_server.utils.gptq.exllamav2 import QuantLinear as exllamav2QuantLinear
except ImportError:
    HAS_EXLLAMA_V2 = False

HAS_EXLLAMA_V1 = False
if not HAS_EXLLAMA_V2:
    HAS_EXLLAMA_V1 = True
    try:
        from lorax_server.utils.gptq.exllama import Ex4bitLinear
    except ImportError:
        HAS_EXLLAMA_V1 = False

...

if use_exllama:
    if HAS_EXLLAMA_V2:
            linear = exllamav2QuantLinear(qweight, qzeros, scales, g_idx, bias, bits, groupsize)
    else:
            linear = Ex4bitLinear(qweight, qzeros, scales, g_idx, bias, bits, groupsize)
```

Happy to defer to your judgement if you think it wouldn't make sense (in which case, we may wish to remove the v1 kernels from the repo and build process).

Reply:
As far I can see there is no problem with older architectures.
Will take a deeper look, preferring to remove v1 to decrease docker size and maintainability